include/pagmo/problems/cec2013.hpp

```diff
@@ -0,0 +1,1218 @@
+                f[0] += -100.0;
+                break;
+            case 15:
+                schwefel_func(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 100.0;
+                break;
+            case 16:
+                katsuura_func(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 200.0;
+                break;
+            case 17:
+                bi_rastrigin_func(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 0);
+                f[0] += 300.0;
+                break;
+            case 18:
+                bi_rastrigin_func(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 400.0;
+                break;
+            case 19:
+                grie_rosen_func(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 500.0;
+                break;
+            case 20:
+                escaffer6_func(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 600.0;
+                break;
+            case 21:
+                cf01(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 700.0;
+                break;
+            case 22:
+                cf02(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 0);
+                f[0] += 800.0;
+                break;
+            case 23:
+                cf03(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 900.0;
+                break;
+            case 24:
+                cf04(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 1000.0;
+                break;
+            case 25:
+                cf05(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 1100.0;
+                break;
+            case 26:
+                cf06(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 1200.0;
+                break;
+            case 27:
+                cf07(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 1300.0;
+                break;
+            case 28:
+                cf08(&x[0], &f[0], nx, &m_origin_shift[0], &m_rotation_matrix[0], 1);
+                f[0] += 1400.0;
+                break;
+        }
+        return f;
+    }
+    /// Box-bounds
+    /**
+     * One of the optional methods of any user-defined problem (UDP).
+     * It returns the box-bounds for this UDP.
+     *
+     * @return the lower and upper bounds for each of the decision vector components
+     */
+    std::pair<vector_double, vector_double> get_bounds() const
+    {
+        // all CEC 2013 problems have the same bounds
+        vector_double lb(m_z.size(), -100.);
+        vector_double ub(m_z.size(), 100.);
+        return {lb, ub};
```

Comment:
FYI, you can write ``return {std::move(lb), std::move(ub)};`` to avoid 2 memory allocations. Or even ``return{vector_double(m_z.size(), -100.),vector_double(m_z.size(),100.)};``.